RocketChat/Rocket.Chat.ReactNative
# Regression: Orientation lock on Android not working

## Proposed changes

[Latest versions](https://github.com/wonday/react-native-orientation-locker/issues/144) from `react-nativate-orientation-locker` doesn't seem to respect orientation lock from device. For now we will just go back to the previous version we had.

## Issue(s)	

Closes #3348

## How to test or reproduce

1. Log in
2. Lock orientation on device's OS
3. Turn the device
4. It should respect the orientation lock and stay in portrait mode

## Screenshots

## Types of changes




## Checklist



## Further comments
